metadata-ingestion/tests/integration/snowflake/snowflake_privatelink_golden.json

```diff
@@ -869,12 +2527,30 @@
 },
 {
     "entityType": "dataset",
-    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:snowflake,test_db.test_schema.table_9,PROD)",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:snowflake,test_db.test_schema.table_1,PROD)",
     "changeType": "UPSERT",
     "aspectName": "upstreamLineage",
     "aspect": {
-        "value": "{\"upstreams\": [{\"auditStamp\": {\"time\": 0, \"actor\": \"urn:li:corpuser:unknown\"}, \"dataset\": \"urn:li:dataset:(urn:li:dataPlatform:snowflake,test_db.test_schema.table_2,PROD)\", \"type\": \"TRANSFORMED\"}]}",
-        "contentType": "application/json"
+        "json": {
```

Comment:
This is the main change where `test_db.test_schema.table_1` now has `other_db.other_schema.table_1` as upstream along with its original upstream `test_db.test_schema.table_2`.

All other changes are due to 
1. Convert serialised MCPs to canonical form for better readability
2. Ordering changes - all lineage mcps are emitted at the end after schema ingestion is over.